Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -287,7 +287,22 @@ spec:
     * This format is required in order to use the NetworkAttachmentDefinition across namespaces.
     * In Openshift, to use a NetworkAttachmentDefinition (NAD) across namespaces, the NAD must be deployed in the `default` namespace. The NAD is then referenced with the namespace: `default/rook-public-nw`
 
-#### Known limitations with Multus
+##### Validating Multus configuration
```

Comment:
This tool seems very useful even beyond multus. Could we generalize it to be a network validation tool, not just specifically for multus? For example, it could test mon and osd ports, or any other general configuration required in the [Ceph network config reference](https://docs.ceph.com/en/quincy/rados/configuration/network-config-ref/#network-configuration-reference). For a separate PR of course...

Reply:
I guess potentially, but let's talk about what planning for that would look like and what specifics would be good to validate. One of the implied follow-ups that seems more worthwhile to me than port access is to do load testing. That could be useful for standard networking as well, to make sure the network doesn't crumble under the load of Ceph's replication+client traffic. But we would need some RADOS experts to help us craft a test that creates a reasonable estimate of client+replication traffic.